tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,5 @@ def test_removing_not_found_returns_none():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(vendor.inventory) == 3
+    assert result is None
```

Comment:
Great assertions, I would also suggest checking the inventory contents, since the length being correct doesn't guarantee that the individual elements are still what we expect.